xds/src/main/java/io/grpc/xds/XdsLoadBalancerProvider.java

```diff
@@ -92,51 +120,17 @@ static ConfigOrError parseLoadBalancingConfigPolicy(
     }
   }
 
-  @VisibleForTesting
-  static LbConfig selectFallbackPolicy(
-      Map<String, ?> rawLoadBalancingPolicyConfig, LoadBalancerRegistry lbRegistry) {
-    List<LbConfig> fallbackConfigs = ServiceConfigUtil.unwrapLoadBalancingConfigList(
-        JsonUtil.getListOfObjects(rawLoadBalancingPolicyConfig, "fallbackPolicy"));
-    LbConfig fallbackPolicy = selectSupportedLbPolicy(fallbackConfigs, lbRegistry);
-    return fallbackPolicy == null ? DEFAULT_FALLBACK_POLICY : fallbackPolicy;
-  }
-
-  @Nullable
-  @VisibleForTesting
-  static LbConfig selectChildPolicy(
-      Map<String, ?> rawLoadBalancingPolicyConfig, LoadBalancerRegistry lbRegistry) {
-    List<LbConfig> childConfigs = ServiceConfigUtil.unwrapLoadBalancingConfigList(
-        JsonUtil.getListOfObjects(rawLoadBalancingPolicyConfig, "childPolicy"));
-    return selectSupportedLbPolicy(childConfigs, lbRegistry);
-  }
-
-  @Nullable
-  private static LbConfig selectSupportedLbPolicy(
-      @Nullable List<LbConfig> lbConfigs, LoadBalancerRegistry lbRegistry) {
-    if (lbConfigs == null) {
-      return null;
-    }
-    for (LbConfig lbConfig : lbConfigs) {
-      String lbPolicy = lbConfig.getPolicyName();
-      if (lbRegistry.getProvider(lbPolicy) != null) {
-        return lbConfig;
-      }
-    }
-    return null;
-  }
-
   /**
    * Represents a successfully parsed and validated LoadBalancingConfig for XDS.
    */
   static final class XdsConfig {
     // FIXME(chengyuanzhang): make cluster name required.
     @Nullable
     final String cluster;
-    // TODO(carl-mastrangelo): make these Object's containing the fully parsed child configs.
     @Nullable
-    final LbConfig childPolicy;
+    final PolicySelection childPolicy;
     @Nullable
-    final LbConfig fallbackPolicy;
+    final PolicySelection fallbackPolicy;
```

Comment:
Do we always have a default fallback policy? If it is, this would be non-null.

Reply:
Not clear. Currently the CdsLoadBalancer will not create a fallback policy in EDS policy. So it is nullable.

Reply:
Hmm, that's bad. We might want to define `EdsConfig` in `EdsLoadBalancerProvider` so `XdsConfig` becomes

```java
class XdsConfig {
  PolicySelection edsPolicy;
  PolicySelection fallbackPolicy;
}
```

Reply:
No, we don't know if we eventually will have FallbackPolicy in EdsConfig.